frontend/src/entry/model/settings/EntryRolesInfo.tsx

```diff
@@ -34,9 +35,23 @@ export default function EntryRolesInfo({ entry }: EntryRolesInfoProps) {
     [entryRoles],
   )
 
-  const entryRolesList = useMemo(() => getFilteredRoles(RoleKind.ENTRY), [getFilteredRoles])
   const schemaRolesList = useMemo(() => getFilteredRoles(RoleKind.SCHEMA), [getFilteredRoles])
 
+  const permissionTableRows = useMemo(() => {
+    return rows.map((row) => (
+      <TableRow key={row.permission} sx={{ '&:last-child td, &:last-child th': { border: 0 } }}>
+        <TableCell component='th' scope='row'>
+          {row.permission}
+        </TableCell>
+        <TableCell align='center'>{row.roles.includes('owner') ? <Done /> : <></>}</TableCell>
```

Comment:
Again, would be good to be using the same RoleValue "enum" for these strings 'owner', etc